lib/biokbase/ServiceWizard/Impl.py

```diff
@@ -127,27 +133,75 @@ def create_compose_files(self, module_version, secure_param_list):
             param_name = secure_param['param_name']
             param_value = secure_param['param_value']
             environ_map['KBASE_SECURE_CONFIG_PARAM_' + param_name] = param_value
+
         
         docker_compose[service_name] = {
                 "image" : module_version['docker_img_name'],
                 "labels" : {
+                    'us.kbase.dynamicservice':"True",
                     'us.kbase.module.version':module_version['version'],
                     'us.kbase.module.git_commit_hash':module_version['git_commit_hash']
                 },
                 "environment" : environ_map
             }
 
+        if len(volume_mounts) > 0:
+            mounts = []
+            for vol in volume_mounts:
+                if vol['read_only']>0:
+                    mounts.append('%s:%s:ro' % (vol['host_dir'], vol['container_dir']))
+                else:
+                    mounts.append('%s:%s' % (vol['host_dir'], vol['container_dir']))
+            docker_compose[service_name]['volumes'] = mounts
+
         rancher_compose[service_name] = {
                 "scale" : 1
             }
 
         return docker_compose, rancher_compose, is_new_stack
 
+    def create_stack(self, service):
```

Comment:
Github may be mangling the diff.  I want to look at it more.

Reply:
It seems like it was fine.  It looks like this new method just factors out some stuff from start() and adds the volume mount code?

Reply:
Dunno, sorry